modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -226,6 +226,12 @@ HTTP messages not containing this header field will be rejected. Two common HTTP
 *Default value*: Blank and unchecked.
 
 |=== 
++
+If you are using WebSockets:
++
+* Provide the correct request headers to upgrade the HTTP connection to WebSockets.
```

Comment:
Does this mean A or B?
A: "Upgrade the HTTP connection to WebSockets by providing the correct request headers."
B: "Provide the correct request headers, so that you can upgrade the HTTP connection to WebSockets."